test/integration_test.py

```diff
@@ -162,7 +162,7 @@ def test_discrete_cnn_env(fast_config: dict, env_name: str) -> None:
 @pytest.mark.parametrize("env_name", continuous_envs)
 def test_continuous_env(fast_config: dict, env_name: str) -> None:
     """Test all continuous envs on random systems."""
-    system_path = random.choice(ppo_systems + sac_systems)
+    system_path = random.choice(ppo_systems + sac_systems + transformer_systems)
```

Comment:
Honestly these systems take long enough, I think we can probably leave them out of here especially because `test_transformer_system` has cont in it
```suggestion
    system_path = random.choice(ppo_systems + sac_systems)
```

Reply:
Yep I agree I'll revert it 🙌